src/rgw/rgw_op.h

```diff
@@ -1773,9 +1773,8 @@ class RGWGetObjLayout : public RGWOp {
   rgw_raw_obj head_obj;
 
 public:
-  RGWGetObjLayout() {
-    delete target;
-  }
+  RGWGetObjLayout() = default;
+  ~RGWGetObjLayout() = default;
```

Comment:
can remove these declarations altogether, the compiler will still generate them

also, please remove the unused member variable `RGWRados::Object *target{nullptr};`